src/components/ApplicationMenu/ApplicationMenu.js

```diff
@@ -241,9 +269,27 @@ const mapDispatchToProps = {
   clearConsole: actions.clearConsole,
   showDeleteProjectPrompt: actions.showDeleteProjectPrompt,
   showResetStatePrompt: actions.showResetStatePrompt,
+  showProjectSettings: actions.showProjectSettings,
+  selectProject: actions.selectProject,
 };
 
 export default connect(
   mapStateToProps,
   mapDispatchToProps
 )(ApplicationMenu);
+
+// helpers
+export const createProjectSelectionSubmenu = (
```

Comment:
could this go before all the redux stuff?

Reply:
sure, I'll move it.